servers/express.js

```diff
@@ -18,85 +18,15 @@ const argvOptions = {
 const argv = require('minimist')(process.argv.slice(2), argvOptions);
 
 // The app
-const app = express();
 const pathToLogFile = argv['logfile'];
-const htmlTemplate = fs.readFileSync(path.join(__dirname, 'pages/templatized.html'), 'utf8');
+const app = express();
 
 app.get('/eventloop', function(req, res) {
-  const chartOpt = makeChartOptions({title: 'Eventloop lag percentiles (ms)', subtitle: '@contrast/route-metrics'});
-  const datarows = makeDataRows(eventloopDataRows);
-  const columns  = makeEventloopColumnNames();
-  
-  const html = populateTemplate(htmlTemplate, chartOpt, columns, datarows);
-  res.send(html);
-});
-
-app.get('/memory', function(req, res) {
-  const chartOpt = makeChartOptions({title: 'Memory usage (megabytes)', subtitle: '@contrast/route-metrics'});
-  const datarows = makeDataRows(memoryDataRows);
-  const columns  = makeMemoryColumnNames();
-  
-  const html = populateTemplate(htmlTemplate, chartOpt, columns, datarows);
-  res.send(html);
+  // wip; got to add query params
```

Comment:
this shouldn't be called the `/eventloop` route when it gets all the data.

maybe `/api` or `/timeseries`?